main.py

```diff
@@ -20,12 +21,18 @@ def __init__(self):
 
         self.selector = CharacterSelector(self.createPlayableCharacters())
         self.selector.onSelect(self.userSelectedCharacter)
+
+        self.crawl = Crawl()
+
         self.fightScreen = None
 
         self.stack = QStackedWidget()
         self.stack.addWidget(self.startMenu)
         self.stack.addWidget(self.selector)
 
+        self.stack.addWidget(self.fightScreen)
+        self.stack.addWidget(self.crawl)
```

Comment:
When do we want to show the crawl? Before the character selection?

Reply:
This kinda depends on the route we take with the story. In my version of the story, it actually makes sense to have the crawl be the very first thing we see which then transitions to the main menu, but it could go right after we click "Start Game"

Reply:
Yeah I was thinking we would show
Start screen -> Crawl -> Character Selector

Reply:
Works for me. Do I need to change the order in the stack or is this more of "Planning for later" type thing?

Reply:
Order shouldn't matter.